### ISSUE TITLE
The Hessian Penalty:A Weak Prior for Unsupervised Disentanglement

### ISSUE THREAD

**Issue by AkiraTOSEI** [AkiraTOSEI/ML_papers#65]
# TL;DR
A study of obtaining a disentangled representation in the latent space of a GAN. Proposed a regularization term using Hessian so that a change in one direction i does not cause a change in other direction j(≠i). Fine-tuning is also possible for trained models.
<img width="953" alt="Baseline ProGAN" src="https://user-images.githubusercontent.com/24524018/91972930-5bb85980-ed56-11ea-9d76-7ba12b7d43aa.png">

# Why it matters:

# Paper URL
https://arxiv.org/abs/2008.10599

# Submission Dates(yyyy/mm/dd)
2020/08/24

# Authors and institutions
William Peebles      John Peebles      Jun-Yan Zhu      Alexei Efros      Antonio Torralba 
- UC Berkeley 
- Yale 
- Adobe Research 
- MIT CSAIL

# Methods

# Results

# Comments
ECCV2020
